### PR TITLE
Code cleanup and alignment tweaks

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

Code cleanup and code formatting!

### How did you do this?

Mostly CSS!

### Why are you choosing this approach?

Better code and alignment on the page.

### Any open questions you want to ask code reviewers?

Nope!

### List of changes:

  - Add wrapper to footer
  - Add wrapper to FAQ
  - Add wrapper to hacker testimonials
  - Add social meta tags
  - Various code formatting

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit d7c9339b25e380a0a11ce10a60c6c699718e0e64

https://deploy-preview-164--qhacks-website.netlify.com

File: src/components/Landing.js [hunk 1]
```diff
@@ -104,11 +95,13 @@ const Landing = () => (
             Queen’s University • February 1-3, 2019
           </h2>
           <img
-            data-cy="mlh-banner"
-            src={banner}
+            data-cy="mlh-logo"
```
Comment (joeytepp) [inline src/components/Landing.js]: Need to update this in cypress too!

File: src/components/FAQs.js [hunk 2]
```diff
@@ -60,11 +60,12 @@ const linkCss = {
...
 
 const FAQs = () => (
-  <div id="faq" css={{ backgroundColor: "#ffffff" }}>
+  <section id="faq" css={{ backgroundColor: "#ffffff" }}>
     <ContentWrapper>
       <div
         css={{
-          padding: "110px 180px 114px 180px",
+          padding: "110px 0px 114px 0px",
+          // screen sizes larger smaller than 780px
```
Comment (joeytepp) [inline src/components/FAQs.js]: Slightly confusing comment here 😕

File: src/__tests__/components/Footer/__snapshots__/Footer.test.js.snap [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Footer renders correctly 1`] = `
-<div
+<footer
```
Comment (joeytepp) [inline src/__tests__/components/Footer/__snapshots__/Footer.test.js.snap]: Nice semantics here 👌

**Comment by RobertWSaunders** [comment]
@rosslh and I talked over Slack and he is going to cover some of the CSS changes I was proposing here in his PR #162. I am going to make some smaller PRs for some of the other changes I made though! Will close this when those PRs get created!

**Comment by joeytepp** [comment]
Sounds good, was worried about merge conflicts between the two PRs

**Pull request closed without merging**